sys-auth/AusweisApp2/metadata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<!-- maintainer-needed -->
```

Comment:
So you won't maintain that package? Otherplase please add yourself including proxy-maint project.

Reply:
I am not a maintainer, not even a proxied.  So I would be glad if you took it for now.
Would I need to become a proxied for putting my name there?

As you have hardware to test, your tests will be highly appreciated.

Reply:
We don't add packages without maintainer.

Reply:
Well, do you want to maintain that package? If so, please add yourself as maintainer into metadata.xml.

Reply:
@ConiKost 
No. Please take it. Or did I misread your offer?

Reply:
Just added myself as a proxied maintainer.  
Anything else to be done to become a PM?